_includes/figure.liquid

```diff
@@ -16,7 +16,7 @@
     {% if site.imagemagick.enabled %}
       <source
         class="responsive-img-srcset"
-        {% if ext == '.jpg' or ext == '.jpeg' or ext == '.png' or ext == '.tiff' or ext == '.gif' %}
+        {% if ext == 'jpg' or ext == 'jpeg' or ext == 'png' or ext == 'tiff' or ext == 'gif' %}
```

Comment:
Thanks for the fix. Can you also move the extensions to be in alphabetical order?

Reply:
```suggestion
        {% if ext == 'gif' or ext == 'jpeg' or ext == 'jpg' or ext == 'png' or ext == 'tiff' %}​⬤
```